db-async-common/src/main/java/com/github/jasync/sql/db/util/ExecutorServiceUtils.kt

```diff
@@ -10,7 +10,7 @@ object ExecutorServiceUtils {
   //creating a lot of threads from that pool means we are not really reactive
   val CachedThreadPool: ExecutorService = Executors.newCachedThreadPool(DaemonThreadsFactory("db-sql-default"))
   val CommonPool: ExecutorService = ForkJoinPool.commonPool()
-  val CommonPoolNetty: ExecutorService = io.netty.util.internal.chmv8.ForkJoinPool.commonPool()
```

Comment:
package was removed in latest version of netty

Reply:
We should probably upgrade netty to latest